### DISCUSSION TITLE
Aula debug do modulo .net - ERRO NA HORA DE EXECUTAR - vscode e .net 8.0

### DISCUSSION THREAD

**Discussion by raulmtsdev** [balta-io/forum#483] [Tira Dúvidas] [answered]
Estou tentando executar o debug mas na hora de executar aparece a seguinte mensagem 

couldn't find a debug adapter desciptor for debug type 'coreclr' (extension might have failed to activate)

ja instalei as extensões mas mesmo assim da a mensagem. 

não sei se é alguma configuracação no launch ou tasks!

--- configuração launch ---

{
    // Use IntelliSense to learn about possible attributes.
    // Hover to view descriptions of existing attributes.
    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
    "version": "0.2.0",
    "configurations": [
        {
            "name": ".NET Core Launch (console)",
            "type": "coreclr",
            "request": "launch",
            "preLaunchTask": "build",
            "program": "${workspaceFolder}/bin/Debug/<target-framework>/<project-name.dll>",
            "args": [],
            "cwd": "${workspaceFolder}",
            "stopAtEntry": false,
            "console": "internalConsole"
        }
    ]
}

---configuração tasks ---

{
    // See https://go.microsoft.com/fwlink/?LinkId=733558
    // for the documentation about the tasks.json format
    "version": "2.0.0",
    "tasks": [
        {
            "label": "build",
            "command": "dotnet",
            "type": "shell",
            "args": [
                "build",
                // Ask dotnet build to generate full paths for file names.
                "/property:GenerateFullPaths=true",
                // Do not generate summary otherwise it leads to duplicate errors in Problems panel
                "/consoleloggerparameters:NoSummary"
            ],
            "group": "build",
            "presentation": {
                "reveal": "silent"
            },
            "problemMatcher": "$msCompile"
        }
    ]
}

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @raulmtsdev, como vai?

Qual versão do .NET/VS e quais extensões está utilizando?

**Reply by raulmtsdev** [reply]
Olá, Professor! Tudo bem? Estou bem graças a Deus. Então eu estou utilizando as versões atuais, no caso o .net 8.0 e o vscode 1.86!

**Reply by raulmtsdev** [reply]
As extensões, instalei as que apareciam para mim de C# da microsoft!

**Reply by andrebaltieri** [reply]
Bom dia, @raulmtsdev, como vai?

Pelo que pesquisei era um problema conhecido, mas que foi corrigido na versão 1.86:
https://github.com/dotnet/vscode-csharp/issues/6030

Não tem nenhuma extensão para ser atualizada?
